GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -869,99 +876,89 @@ - (void)testNotHandleWrongPath {
 #pragma mark - Tests - disconnectWithCallback:
 
 // Verifies disconnect calls callback with no errors if access token is present.
-- (void)testDisconnect_accessToken {
+- (void)testDisconnect_accessTokenIsPresent {
```

Comment:
I see; this is where you got the `_` in the test names for `GIDHTTPFetcherTest`. IMO, that underscore isn't needed as ObjC typically uses camel-case. I suggest we migrate to that style as we update existing tests and also use camel-case when we write new tests.